webpack.config.js

```diff
@@ -175,7 +189,55 @@ if (TARGET !== 'readme') {
         loaderConfig: require("./UI/js-src/lsmb/webpack.loaderConfig.js"),
         environment: { dojoRoot: "UI/js" }, // used at run time for non-packed resources (e.g. blank.gif)
         buildEnvironment: { dojoRoot: "node_modules" }, // used at build time
-        locales: ["en"],
+        locales: [
```

Comment:
This is the current list of supported languages (should that be called locales?); shouldn't we make that dynamic?

Reply:
Fixed